blocks/autocompleters/index.js

```diff
@@ -102,8 +102,8 @@ export function blockAutocompleter( { onReplace } ) {
  * @returns {Completer} Completer object used by the Autocomplete component.
  */
 export function userAutocompleter() {
-	const getOptions = () => {
-		return ( new wp.api.collections.Users() ).fetch().then( ( users ) => {
+	const getOptions = ( search = '' ) => {
+		return ( new wp.api.collections.Users() ).fetch( { data: { search: search } } ).then( ( users ) => {
```

Comment:
You can use ES6 shorthand for `search` field:
```js
{ data: { search }
```

Reply:
Good point, changed